### PR TITLE
Docs restructuring

### PR DESCRIPTION
**Pull request by mikemonteith**
- Render nunjucks tags inside markdown files
- Simplify file structure
- Use markdown for content pages
- Add a 'plain' layout with title and content
- Recompile docs when our source changes (useful during development of own component nunjucks macros)

### CODE REVIEW

File: docs/_includes/layouts/plain.njk [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% extends "./base.njk" %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">
```
Comment (davidhunter08) [inline docs/_includes/layouts/plain.njk:4]: Should be `nhsuk-heading-xl`

**Review by davidhunter08 — CHANGES_REQUESTED** [review]
Looks good. Just need the heading class name changing.

File: docs/_includes/layouts/plain.njk (hunk 1 above)
Comment (mikemonteith) [reply docs/_includes/layouts/plain.njk]: I've removed the class completely

**Review by davidhunter08 — APPROVED** [review]
LGTM

**Pull request merged**